use-timescale/data-tiering/creating-data-tiering-policy.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Creating a Data Tiering Policy
+excerpt: How to create a data tiering policy
+product: [ cloud ]
+keywords: [ data tiering, tiering ]
+tags: [ storage, data management ]
+---
+
+# Creating a data tiering policy
+
+To automate archival of historical data, create a data tiering policy that
+automatically moves data to object storage. Any chunks that only contain data
+older than the `move_after` threshold are moved. This works similarly to a
+[data retention policy][data-retention], but chunks are moved rather than deleted.
+
+The data tiering policy schedules a job that runs periodically to migrate
+eligible chunks. The migration is asynchronous.
```

Comment:
The chunks are tiered once they appear in the `timescaledb_osm.tiered_chunks` view. Tiering does not influence your ability to query the chunks.

Reply:
This was copied from the existing documentation. Feel free to update.